### ISSUE TITLE
UV index and a few other conditons in the forecast

### ISSUE THREAD

**Issue by wigster** [jaydeethree/Home-Assistant-weatherdotcom#27]
Hi,

In 1.1.5 you filled out all the possible fields in the current conditions weather entity.

Would it be possible for you to do the same for the forecasts? The TWC forecast API seems to provide data for all the possible HA forecast attributes. The missing ones are:

- humidity: relativeHumidity
- native_apparent_temperature: temperatureFeelsLike
- native_dew_point: temperatureDewPoint
- native_pressure: pressureMeanSeaLevel  
- native_wind_gust_speed: windGust
- uv_index: uvIndex

The only comment/complication is that the pressure in the forecast is only at sea level, while in the current conditions you pick up the pressureAltimeter key which is actually observed at ground level. If you wanted to get technical, this is a simple conversion for any realistic situation, which is given by the formula for P from [here](https://www.sensorsone.com/elevation-station-qfe-sea-level-qnh-pressure-calculator/)

The problem then would be getting the elevation of the location, but in principle it is possible from somewhere like [open-elevation](https://open-elevation.com/)

Anyway, thanks for creating this. The richness of the data has really made many things possible that until now I've been kludging from multiple sources.

**Comment by jaydeethree** [comment]
Thanks for the detailed issue report! I think I'll have time to add all of this next week.

**Comment by jaydeethree** [comment]
This was completed in https://github.com/jaydeethree/Home-Assistant-weatherdotcom/commit/6e40e184f050f23075b348f60af8f2c04ac0c60b and is available in version 1.1.6.

Some notes:

* I did not add pressure data to the forecast sensors. As you noted, Weather.com only provides forecast pressure data at sea level, which may not be accurate for people who are at higher elevations. Programmatically determining elevation is not something I want to deal with in this integration, and I'd rather have missing data than potentially inaccurate data.

* Apparent temperature, dew point, and wind gust data are not provided by the daily forecast API - they only exist in the hourly forecast API. I've added that data for hourly forecasts but it will not be present in daily forecasts.
